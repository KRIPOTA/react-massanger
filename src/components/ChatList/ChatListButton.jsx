import Button from '@mui/material/Button';


export function ChatListButton() {
    return (
        <>
            <Button variant="contained" color="success" type="submit" size="small" sx={{ marginLeft: '3px', marginTop: '14px', width: '150px' }}></Button>
        </>
    )
}