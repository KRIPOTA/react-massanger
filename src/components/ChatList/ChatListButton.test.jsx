import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom'

import { ChatListButton } from './ChatListButton'

describe('testing ChatListButton', () => {
    it('render component ChatListButton', () => {
        render(<ChatListButton>Create chat</ChatListButton>)
    })

    it('render ChatListButton with snapshot', () => {
        const { asFragment } = render(<ChatListButton>Change name</ChatListButton>)
        expect(asFragment()).toMatchSnapshot()
    })
})
