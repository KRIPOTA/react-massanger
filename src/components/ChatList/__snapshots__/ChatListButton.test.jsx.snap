// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Button render with snapshot 1`] = `
<DocumentFragment>
  <button
    class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSuccess MuiButton-sizeSmall MuiButton-containedSizeSmall MuiButton-root MuiButton-contained MuiButton-containedSuccess MuiButton-sizeSmall MuiButton-containedSizeSmall css-i6qjk0-MuiButtonBase-root-MuiButton-root"
    tabindex="0"
    type="submit"
  >
    <span
      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
    />
  </button>
</DocumentFragment>
`;

exports[`testing ChatListButton render ChatListButton with snapshot 1`] = `
<DocumentFragment>
  <button
    class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSuccess MuiButton-sizeSmall MuiButton-containedSizeSmall MuiButton-root MuiButton-contained MuiButton-containedSuccess MuiButton-sizeSmall MuiButton-containedSizeSmall css-i6qjk0-MuiButtonBase-root-MuiButton-root"
    tabindex="0"
    type="submit"
  >
    <span
      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
    />
  </button>
</DocumentFragment>
`;
